### DISCUSSION TITLE
數據外洩

### DISCUSSION THREAD

**Discussion by AngusGuei** [1101-GS3073-C/Python-Community-GS3073-C#416] [General]
現在不只要防外面的攻擊，還要防備員工手賤...

https://technews.tw/2022/05/25/data-loss-prevention-on-email-in-2022-report/

**Comment by luo910227** [comment]
數據外洩真的是個甚難防範的問題，大多也是員工造成，像我爸就因為博客萊個資外洩間接被騙了很多錢
